### ISSUE TITLE
Pyston build for ubuntu 22.04

### ISSUE THREAD

**Issue by stuaxo** [pyston/pyston_v2#204]
Hi,
   Now Ubuntu 22.04 is out, is it possible to provide release a deb for it ?
(21.10 might be nice too)

Cheers
Stuart

**Comment by undingen** [comment]
good idea, I will create 22.04 packages. We don't plan to create packages for non LTS versions right now so for 21.10 I recommend using the "portable release" archive.

**Comment by stuaxo** [comment]
Cheers :)
